packages/vscode-extension/src/commands/CreateProject.ts

```diff
@@ -3,255 +3,267 @@
-                 {
-                     forcenewwindow: openProjectInNewWindow ? CREATE_PROJECT.BUTTONS.NEW_WINDOW : false,
-                 }
-             );
-         }
-     }
- 
-     private async getCommandArgs(): Promise<{ [key: string]: string }> {
-         const commandArgs: { [key: string]: string } = {};
- 
-         const selectedFolder = await window.showOpenDialog({
-             canSelectFiles: false,
-             canSelectFolders: true,
-             canSelectMany: false,
-             title: this.translationService.getMessage(CREATE_PROJECT.SELECT_PARENT_FOLDER),
-             defaultUri: Uri.file(os.homedir()),
-         });
-         if (!selectedFolder) {
-             return {};
-         }
- 
-         let projectType = await this.promptSelectProjectTypeQuestion();
-         if (!projectType) {
-             return {};
-         }
+    PUBLISHER_ID: 'publisherid',
+    TYPE: 'type',
+};
+const VSCODE_OPEN_FOLDER_COMMAND = 'vscode.openFolder';
+
+export default class CreateProject extends BaseAction {
+
+    constructor() {
+        super(COMMAND_NAME);
+    }
+
+    protected async execute(): Promise<void> {
+        const commandArgs = await this.getCommandArgs();
+        if (Object.keys(commandArgs).length === 0) {
+            return;
+        }
+
+        output.show(true);
+
+        const runSuiteCloudCommandPromise = this.runSuiteCloudCommand(commandArgs, commandArgs.parentdirectory);
+        this.messageService.showInformationMessage(
+            this.translationService.getMessage(COMMAND.TRIGGERED, this.vscodeCommandName),
+            this.translationService.getMessage(CREATE_PROJECT.MESSAGES.CREATING_PROJECT),
+            runSuiteCloudCommandPromise,
+            false
+        );
+        const actionResult = await runSuiteCloudCommandPromise;
+        if (actionResult.isSuccess()) {
+            await this.openProjectInNewWindow(actionResult.projectDirectory);
+        } else {
+            this.messageService.showCommandError(undefined, false);
+        }
+    }
+
+    protected validate(): { valid: false; message: string } | { valid: true } {
+        return {
+            valid: true,
+        };
+    }
+
+    private async openProjectInNewWindow(projectAbsolutePath: string): Promise<void> {
+        const openProjectInNewWindow = await window.showInformationMessage(
+            this.translationService.getMessage(CREATE_PROJECT.MESSAGES.OPEN_PROJECT),
+            this.translationService.getMessage(CREATE_PROJECT.BUTTONS.THIS_WINDOW),
+            this.translationService.getMessage(CREATE_PROJECT.BUTTONS.NEW_WINDOW),
+        );
+
+        if (openProjectInNewWindow !== undefined) {
+            commands.executeCommand(
+                VSCODE_OPEN_FOLDER_COMMAND,
+                Uri.file(projectAbsolutePath),
+                {
+                    forcenewwindow: openProjectInNewWindow ? this.translationService.getMessage(CREATE_PROJECT.BUTTONS.NEW_WINDOW) : false,
+                }
```

Comment:
I think this should be:
`forcenewwindow: openProjectInNewWindow === this.translationService.getMessage(CREATE_PROJECT.BUTTONS.NEW_WINDOW)`